sunpy/net/dataretriever/sources/adapt.py

```diff
@@ -0,0 +1,74 @@
+# # Author: Gilly <gilly@swri.org>
+# import sunpy.net
+
```

Comment:
```suggestion
```
The plan (but not yet formal) has been to try and not have authorship information.
But move this information into our formal documentation to highlight people who have contributed heavily to parts of sunpy.